src/components/common/Tabs/ChatTab/index.js

```diff
@@ -0,0 +1,91 @@
+import Back from 'components/common/Back';
+import useTranslation from 'hooks/useTranslation';
+import { InputText } from 'primereact/inputtext';
+import { useEffect, useRef, useState } from 'react';
+import { ActionCableConsumer } from 'react-actioncable-provider';
+import { useDispatch, useSelector } from 'react-redux';
+import { useGetConversationsQuery, useGetMessagesQuery } from 'services/model/conversations';
+import { selectTargets } from 'services/model/targets';
+import { setHomeTab } from 'state/slices/tabSlice';
+import { tabsEnum } from '..';
+import './styles.scss';
+
+const ChatTab = () => {
+  const t = useTranslation();
+  const dispatch = useDispatch();
+  const cableRef = useRef(null);
+  const { conversationSelected: conversationId } = useSelector(selectTargets);
+  const { data: conversations } = useGetConversationsQuery();
+  const [match, setMatch] = useState({});
+  const { data: dataMessages } = useGetMessagesQuery({ conversationId });
+  const messages = dataMessages?.messages || [];
+  const [message, setMessage] = useState('');
+
+  useEffect(() => {
+    if (conversations?.matches) {
+      setMatch(conversations.matches.find(m => m.match_id === conversationId) || {});
+    }
+  }, [conversationId, conversations]);
+
+  const onBack = () => {
+    dispatch(setHomeTab(tabsEnum.profile));
+  };
+
+  const handleReceivedChat = data => {
+    console.log('recibi un mensaje', data);
```

Comment:
And remember to delete those logs after

Reply:
Yeah sadly this is from an incomplete task that I had trouble with and I'll need to finish in the future (actioncable)

Reply:
Sure, don't worry!